### PR TITLE
feat: Updates sidebar to accomodate sub menu 

### PR DESCRIPTION
**Pull request by nithindavid**
# Pull Request Template

## Description

image

image

image

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### CODE REVIEW

File: app/javascript/dashboard/i18n/sidebarItems/campaigns.js [hunk 1]
```diff
@@ -2,29 +2,24 @@ import { frontendURL } from '../../helper/URLHelper';
...
       icon: 'ion-arrow-swap',
       label: 'ONGOING',
+      key: 'ongoingCampaigns',
       hasSubMenu: false,
       toState: frontendURL(`accounts/${accountId}/campaigns/ongoing`),
       toStateName: 'settings_account_campaigns',
     },
-    onOffCampaigns: {
+    {
+      key: 'onOffCampaigns',
```
Comment (iamsivin) [inline app/javascript/dashboard/i18n/sidebarItems/campaigns.js:15]:
```suggestion
      key: 'oneOffCampaigns',
```

File: app/javascript/shared/components/ui/dropdown/DropdownDivider.vue [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  ></li>
+</template>
+<script>
+export default {};
+</script>
+<style lang="scss" scoped>
+.dropdown-menu--divider {
+  list-style: none;
+  margin: var(--space-small) 0;
+
```
Comment (iamsivin) [inline app/javascript/shared/components/ui/dropdown/DropdownDivider.vue:15]:
```suggestion

```

**Review by iamsivin — COMMENTED** [review]
Nice work 🔥. I have tested this branch local I see an issue with the Chatwoot logo and inbox name, you can see it in the screen record that I have attached.


https://user-images.githubusercontent.com/64252451/143530897-b4d24beb-d59f-4155-b05a-ef84f626a307.mp4

File: app/javascript/dashboard/components/layout/AvailabilityStatus.vue [hunk 3]
```diff
@@ -1,66 +1,41 @@
...
-                <availability-status-badge :status="status.value" />
-                {{ status.label }}
-              </woot-button>
-            </woot-dropdown-item>
-          </woot-dropdown-menu>
-        </div>
-      </transition>
-
+  <woot-dropdown-menu>
+    <woot-dropdown-header title="Set yourself as" />
```
Comment (muhsin-k) [inline app/javascript/dashboard/components/layout/AvailabilityStatus.vue:3]: i18n

File: app/javascript/dashboard/components/layout/sidebarComponents/AgentDetails.vue [hunk 4]
```diff
@@ -25,39 +25,27 @@ export default {
   computed: {
     ...mapGetters({
       currentUser: 'getCurrentUser',
+      allAgents: 'agents/getAgents',
```
Comment (muhsin-k) [inline app/javascript/dashboard/components/layout/sidebarComponents/AgentDetails.vue:28]: Unused computed property

File: app/javascript/dashboard/components/layout/sidebarComponents/AgentDetails.vue [hunk 5]
```diff
@@ -25,39 +25,27 @@ export default {
...
       currentRole: 'getCurrentRole',
```
Comment (muhsin-k) [inline app/javascript/dashboard/components/layout/sidebarComponents/AgentDetails.vue:29]: Unused computed property

File: app/javascript/dashboard/modules/sidebar/mixins/routes.mixin.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
+import { getSidebarItems } from 'dashboard/i18n/default-sidebar';
+
+export default {
+  computed: {
+    currentRoute() {
+      return this.$store.state.route.name;
+    },
+    sidemenuItems() {
```
Comment (muhsin-k) [inline app/javascript/dashboard/modules/sidebar/mixins/routes.mixin.js:8]:
```suggestion
    sideMenuItems() {
```

File: app/javascript/dashboard/components/layout/sidebarComponents/AgentDetails.vue (hunk 5 above)
Comment (muhsin-k) [inline app/javascript/dashboard/components/layout/sidebarComponents/AgentDetails.vue:29]: Unused computed property

**Comment by muhsin-k** [comment]
@nithindavid I think we need to add the button "Add label" for creating a new label from the contact page.

Screenshot 2021-11-26 at 1 21 46 PM

**Comment by nithindavid** [comment]
@iamsivin @muhsin-k Thanks for the review, I have updated with the suggestions.

**Comment by muhsin-k** [comment]
@nithindavid Please fix the specs

**Comment by iamsivin** [comment]
@nithindavid I saw an error throwing when selecting the profile icon which you can see in the below recording.

https://user-images.githubusercontent.com/64252451/143817042-de31515c-5475-49bb-9be4-a06a30d942a2.mov

**Comment by nithindavid** [comment]
@muhsin-k @iamsivin Done!

**Comment by iamsivin** [comment]
@nithindavid Looks good to me 🎉.

**Review by iamsivin — APPROVED** [review]

**Pull request merged**